docs/methods/search_parameter_optimization.md

```diff
@@ -0,0 +1,131 @@
+# Search Parameter Optimization
+
+## Calibration and optimization
+### Overall process
+The first step of every alphaDIA run is the optimization of search parameters and the calibration of the empirical or fully predicted spectral library to the observed values. This step has two main purposes: 1) removing the systematic deviation of observed and library values, and 2) optimizing the size of the search space to reflect the expected deviation from the library values. For DIA search this means calibration and optimization of certain parameters: retention time, ion mobility, precursor m/z and fragment m/z. The process of iterative calibration and optimization is illustrated below.
```

Comment:
(nit) `AlphaDIA` .. also, there's a typo in `statsitical` few paragraphs below ;-)